docs/source/plugins/03_plugin_structure.rst

```diff
@@ -21,9 +21,8 @@ First, to create the plugin use :program:`new`:
 Pyinvoke tasks
 --------------
 
-Except for creating the plugin structure, all other activities that you might want to perform, you will use ``pyinvoke`` tasks.
+Except for creating the plugin structure, all other activities that you might want to perform uses the following ``pyinvoke`` tasks:
 
-Below there is detailed list on what they are meant to do. However, in general, the tasks that you will use the most are ``compile``,
 ``package`` and ``update``.
```

Comment:
I don't get why we need to remove the last line. You can check that this line fits when completed with the tasks description [in the generated documentation](https://alfasim-sdk.readthedocs.io/en/latest/plugins/03_plugin_structure.html).

However, if you still want to remove it, you need to remove the following line as well

```rst
``package`` and ``update``
```

because it seems to be a leftover

Reply:
You're right. It was a leftover, but I see that it could be kept as well.  I rewrote both paragraphs as 
```
Once the plugin structure is created, you can perform all necessary activities using ``pyinvoke`` tasks, with ``compile``, ``package``  and ``update`` beeing the 
most commonly used during plugin development. Below you can find a list of all available along with their descriptions.
```